### PR TITLE
updated home template to use the same structure

### PR DESCRIPTION
**Pull request by Sandravaphilips**
Screenshot 2024-11-28 at 14 59 18
Screenshot 2024-11-28 at 14 59 05

### CODE REVIEW

**Comment by longhotsummer** [comment]
This is looking good. The old banner with a background is updated to look like this, which looks like it's still missing in this PR:

image

The row of partner logos then goes beneath that.

**Comment by actlikewill** [comment]
also we have some hero text that went over the images for most liis. @longhotsummer are we dropping that text in the new design?
image

**Comment by longhotsummer** [comment]
> also we have some hero text that went over the images for most liis. @longhotsummer are we dropping that text in the new design? image

No, the idea is to preserve the text in the new area but drop the image, and put the logo on the right.

**Comment by Sandravaphilips** [comment]
@longhotsummer @actlikewill 
Screenshot 2024-12-06 at 14 22 53
Screenshot 2024-12-06 at 14 23 08

File: civlii/templates/liiweb/home.html [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
 {% load i18n static %}
-{% block hero-headings %}
-  <h1 class="mb-4">{% trans "Welcome to the Côte d'Ivoire Legal Information Institute" %}</h1>
-  <h4 class="mb-4">
-    {% blocktrans trimmed %}
+{% block hero-headings-title %}
+  {% trans "Welcome to the Côte d'Ivoire Legal Information Institute" %}
+{% endblock %}
+{% block hero-headings-welcome %}
   Free access to the law of Côte d'Ivoire. CIVLII is a member of the community of African LIIs and of the Free Access to Law Movement.
```
Comment (actlikewill) [inline civlii/templates/liiweb/home.html:7]: these need to be translatabale

File: eswatinilii/templates/liiweb/home.html [hunk 2]
```diff
@@ -9,13 +9,11 @@
...
-  <h4 class="mb-4">
-    {% blocktrans trimmed %}
-    Free access to the law of eSwatini. eSwatiniLII is operated by the Judiciary of eSwatini and is a member of the community of African LIIs and of the Free Access to Law Movement.
-  {% endblocktrans %}
-  </h4>
+{% block hero-headings-title %}
+  {% trans 'Welcome to eSwatini Legal Information Institute' %}
+{% endblock %}
+{% block hero-headings-welcome %}
+  Free access to the law of eSwatini. eSwatiniLII is operated by the Judiciary of eSwatini and is a member of the community of African LIIs and of the Free Access to Law Movement.
```
Comment (actlikewill) [inline eswatinilii/templates/liiweb/home.html:16]: translatable

**Review by actlikewill — APPROVED** [review]

**Pull request merged**